### PR TITLE
Added warning in playlist.yml and example playlist

### PR DESCRIPTION
**Pull request by Legoman99573**

### CODE REVIEW

**Comment by Legoman99573** [comment]
added bugfix in

**Comment by Legoman99573** [comment]
added audio command :P

File: Plugin source code/src/main/java/net/openaudiomc/commands/admin/CommandOauth.java [hunk 1]
```diff
@@ -21,7 +21,7 @@ public boolean isPlayerCommand() {
     @Override
     public void execute(CommandSender sender, String[] args) {
         sender.sendMessage(Main.PREFIX + "Generating url.");
-        sender.sendMessage(Main.PREFIX + ChatColor.RED + "Please note! this key can only be used ONCE and should only be used on a site that starts with oauth.openaudiomc.net!");
```
Comment (ApocalypsjeNL) [inline Plugin source code/src/main/java/net/openaudiomc/commands/admin/CommandOauth.java]: Why? AudioLoad also uses this oauth key and is on oauth.openaudiomc.net and not on plus.openaudiomc.net

File: Plugin source code/src/main/java/net/openaudiomc/core/Main.java [hunk 2]
```diff
@@ -227,10 +228,11 @@ public void createPlaylist() {
 
             }
             FileConfiguration datafileInst = YamlConfiguration.loadConfiguration(dataFile);
-            datafileInst.set("Description", "Playlists are stored here");
+            datafileInst.set("Description", "Playlists are stored here. Using YouTube playlist would require 1 song due to a bug client side.");
```
Comment (ApocalypsjeNL) [inline Plugin source code/src/main/java/net/openaudiomc/core/Main.java]: Isn't this bug already fixed?

Comment (Legoman99573) [reply Plugin source code/src/main/java/net/openaudiomc/core/Main.java]: yeah, i will remove it

Comment (Legoman99573) [reply Plugin source code/src/main/java/net/openaudiomc/core/Main.java]: updated it ;)

**Pull request merged**